pages/common/yacas.md

```diff
@@ -0,0 +1,28 @@
+# yacas
+
+> Yet Another Computer Algebra System.
+> More information: <http://www.yacas.org>.
+
+- Start an interactive console session:
+
+`yacas`
+
+- In the console session, execute a statement:
```

Comment:
I've multiple suggestions:
```suggestion
- In the `yacas` session, execute a statement:
```
```suggestion
- While running `yacas`, execute a statement:
```
```suggestion
- Execute a statement within `yacas`:
```
Take the one you like the most (or your own if you have another idea).
That also applies to the next example.

Reply:
I have pushed an update using this idea.